pkg/connector/group.go

```diff
@@ -268,14 +273,19 @@ func (o *groupResourceType) Get(ctx context.Context, resourceId *v2.ResourceId,
 		l.Error("google-workspace: group had no id", zap.String("name", g.Name))
 		return nil, nil, nil
 	}
-	annos := &v2.V1Identifier{
+	var annos annotations.Annotations
+	annos.Update(&v2.V1Identifier{
 		Id: g.Id,
-	}
+	})
+	annos.Update(&v2.RawId{
+		Id: g.Id,
+	})
 	traitOpts := []sdkResource.GroupTraitOption{sdkResource.WithGroupProfile(groupProfile(ctx, g))}
-	groupResource, err := sdkResource.NewGroupResource(g.Name, resourceTypeGroup, g.Id, traitOpts, sdkResource.WithAnnotation(annos))
+	groupResource, err := sdkResource.NewGroupResource(g.Name, resourceTypeGroup, g.Id, traitOpts)
```

Comment:
I think you can use the ResourceOption:

```suggestion
	groupResource, err := sdkResource.NewGroupResource(g.Name, resourceTypeGroup, g.Id, traitOpts, sdkResource.WithAnnotation(annos))
```